### PR TITLE
feat: instagram nft avater

### PR DESCRIPTION
**Pull request by nuanyang233**
## Description

Closes MASK-752 

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Review by guanbinrui — DISMISSED** [review]

**Comment by nuanyang233** [comment]
Plz don't merge this pr before QA testing is completed.

**Comment by developerfred** [comment]
@nuanyang233 I love it

**Comment by wenluomask** [comment]
Checked.

File: packages/mask/src/plugins/Avatar/Services/rss3.ts [hunk 1]
```diff
@@ -23,6 +23,11 @@ export async function getNFTAvatarFromRSS(userId: string, address: string, snsKe
     return result?.nft
 }
 
+export function clearTargetCache(userId: string, address: string, snsKey: RSS3_KEY_SNS) {
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Avatar/Services/rss3.ts]: `clear` seems you'd like to delete all cached data.

```suggestion
export function deleteTargetCache(userId: string, address: string, snsKey: RSS3_KEY_SNS) {
```

File: packages/mask/src/social-network-adaptor/instagram.com/injection/NFT/NFTAvatarEditProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    const { classes } = useStyles()
+
+    if (location.pathname?.includes('/edit')) return null
+
+    return (
+        <>
+            <NFTAvatarButton onClick={onClick} classes={classes} />
+        </>
+    )
```
Comment (guanbinrui) [inline packages/mask/src/social-network-adaptor/instagram.com/injection/NFT/NFTAvatarEditProfile.tsx]:
```suggestion
    return  <NFTAvatarButton onClick={onClick} classes={classes} />
```

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**